partcad-cli/src/partcad_cli/click/commands/convert.py

```
@@ -0,0 +1,138 @@
+import rich_click as click
+import partcad.logging as logging
+import partcad.utils as pc_utils
+from partcad import Project
+from typing import List, Optional, Tuple
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)